reasoner/pom.xml

```diff
@@ -351,6 +351,12 @@
         </exclusions>
       </dependency>
       <!-- hive end -->
+      <dependency>
+        <groupId>com.aliyun.openservices</groupId>
+        <artifactId>log-loghub-producer</artifactId>
+        <version>0.1.4</version>
+      </dependency>
+      <!-- sls end -->
```

Comment:
missing sls start ?

Reply:
What start?  The above dependency is all I need to run.